code/controllers/subsystem/tickets/tickets.dm

```diff
@@ -111,7 +113,43 @@ SUBSYSTEM_DEF(tickets)
 		message_staff("<span class='[span_class]'>[usr.client] / ([usr]) resolved [ticket_name] number [N]</span>")
 		to_chat_safe(returnClient(N), "<span class='[span_class]'>Your [ticket_name] has now been resolved.</span>")
 		return TRUE
+		
 
+/datum/controller/subsystem/tickets/proc/autoRespond(N)
+	var/datum/ticket/T = allTickets[N]
+	var/client/C = usr.client
+	if((T.staffAssigned && T.staffAssigned != C) || (T.lastStaffResponse && T.lastStaffResponse != C) || T.ticketState != TICKET_OPEN) //if someone took this ticket, is it the same admin who is autoresponding? if so, then skip the warning
+		if(alert(usr, "[T.ticketState == TICKET_OPEN ? "This ticket is already marked as closed or resolved" : "Another admin appears to already be handling this."] Are you sure you want to continue?", "Confirmation", "Yes", "No") != "Yes")
+			return
+	T.assignStaff(C)
+	
+	//try to keep this list where more common things are at the top yeah? thx
+	var/response_phrases = list("Cancel" = "Cancel",
+		"Thanks" = "Thanks, have a Paradise day!", 
+		"Already Resolved" = "The problem has been resolved already.",
+		"Mentorhelp" = "Please redirect your question to Mentorhelp, as they are better experienced with these types of questions.",
```

Comment:
Not an English expert, but wouldn't "more suitable to" instead of "better experienced" be better phrasing?

Admin does not necessarily have less experience but mentors are generally the better candidate for gameplay question.

Reply:
They are better experienced, because that's what they do all the time - gameplay mechanic questions - while admins deal with the rules all the time.

Reply:
I'm gonna mark this as resolved so it doesnt take up a bunch of space on the files page but if you want it changed lmk